src/app/modules/form/admin-form/admin-form.payments.controller.ts

```diff
@@ -70,6 +73,9 @@ export const handleConnectAccount: ControllerHandler<{
           level: PermissionLevel.Write,
         }),
       )
+      // Payments should not be allowed to connect
+      // since single submission forms do not support it currently.
+      .andThen(assertFormIsSingleSubmissionDisabled)
```

Comment:
We'll need to also add a `mapRouteError` for the new error (`UnsupportedSettingsError`) that's thrown, otherwise we'll get 5xx errors when user tries to connect (by manipulating the FE) when ISS is `enabled`.

Reply:
thanks for the catch!